tests/e2e/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
     environment:
       ENABLE_CUDA: "0"
   neo4j:
-    image: neo4j:5-enterprise
+    image: neo4j:5.24-enterprise
```

Comment:
I had to fix this to ensure we get the latest Cypher syntax changes

Reply:
Do you know which version was pulled if not the latest? The tag has been updated recently, could that have an impact?

Reply:
I could not reproduce the warnings with `5-enterprise`, so it must be 5.22 or below. After adding this I can reproduce the warnings

Reply:
Which version are we using in CI?

Reply:
(sorry I'm nitpicking on this, mainly because I'm not sure about the implications of pinning the version in e2e)